src/components/cargo/output_channel_task_manager.ts

```diff
@@ -83,6 +83,7 @@ export class OutputChannelTaskManager {
         this.runningTask.setStarted(() => {
             this.channel.clear();
             this.channel.append(`Started cargo ${args.join(' ')}\n\n`);
+            this.diagnosticPublisher.clearDiagnostics();
```

Comment:
Make an empty line between 85 and 86 lines.
I have my inner convention that using different objects should be separated by an empty line.